src/sortStudents.ts

```diff
@@ -1,16 +1,59 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married?: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function calcAvgGrade(arrOfGrades: number[]): number {
+  return arrOfGrades.reduce((x: number, y:number) => {
+    return x + y;
+  }) / arrOfGrades.length;
```

Comment:
```suggestion
  return arrOfGrades.reduce((x: number, y:number) => x + y, 0) / arrOfGrades.length;
```
The result in the callback is too short - U can return it just like this.  Also, I think it`s a good idea to set the initial value of the accumulator)

Reply:
agree